src/resolvers/query/platform-versions/index.js

```diff
@@ -0,0 +1,13 @@
+import fragment from "./fragment";
+import { addFragmentToInfo } from "graphql-binding";
+
+/*
+ * Get list of available platform versions.
+ * @param {Object} parent The result of the parent resolver.
+ * @param {Object} args The graphql arguments.
+ * @param {Object} ctx The graphql context.
+ * @return {[]PlatformVersion} List of PlatformVersions.
+ */
+export default async function platformVersions(parent, args, ctx, info) {
+  return ctx.db.query.platformVersions({}, addFragmentToInfo(info, fragment));
```

Comment:
We shouldn't have `addFragmentToInfo` here. That forces fields to be returned (which we sometimes need on some types for other resovlers) but in this case we should leave it entirely up to the callers control

```suggestion
  return ctx.db.query.platformVersions({}, info);
```

And then we can delete the new fragment.js

Reply:
@andriisoldatenko ^^

Reply:
👍